src/jobservice/period/enqueuer_test.go

```diff
@@ -120,16 +120,26 @@ func (suite *EnqueuerTestSuite) TestEnqueuer() {
 }
 
 func (suite *EnqueuerTestSuite) prepare() {
+	var cronSpec string
+
 	now := time.Now()
 	minute := now.Minute()
 
-	coreSpec := fmt.Sprintf("0-59 %d-%d * * * *", minute, minute+2)
```

Comment:
`coreSpec := fmt.Sprintf("0-59 %d,%d,%d * * * *", minute, (minute+1) % 60, (minute+2) % 60)` will be better than `switch`.